pages/dashboard/v2/details/[id]/[studentEmail].js

```diff
@@ -82,9 +119,17 @@ export default function StudentDetails({
               <Link href={'/'}> Menu</Link>
             </div>
           </Navbar>
-          <h1>
-            {studentEmail}&apos;s progress in {classroomName}
-          </h1>
+          <div className={styles.student_header}>
+            <h1>
+              {studentEmail}&apos;s progress in {classroomName}
+            </h1>
+          </div>
+
+          <DetailsBoard
+            studentCurriculumData={studentCurriculumData}
```

Comment:
we no longer need to pass props here since this is being imported from API processor

Reply:
We will still need props here because [getServerSideProps() ](https://nextjs.org/docs/pages/building-your-application/data-fetching/get-server-side-props) is what gives us access to data stored on the server, which is needed as the reference/parameters for the utility functions in api_processor.js. Even though functions from api_processr.js can be called in the other components, it is still necessary to fetch the data on the server first - and we need getServerSideProps() + context.params for this.